fulp_modules/features/species/beefmen/mob/beefman.dm

```diff
@@ -127,22 +127,23 @@
-		After drawing straws, the Cook was sent to fetch some more meat from the Morgue, unaware of the events that will transpire. \
-		'It's normal for the Kitchen to grab dead bodies, right? It's not like they need them... Right?' The Cook thought, \
-		inattentively grabbing the first body they could find, trying to get this over with before it becomes a memory. \
-		What the Cook hadn't noticed, the Morgue's tray was green, the body was filled with a soul, one that was begging not to be gibbed.",
-
-		"The Cook one'd and two'd the body into the gibber and turned it on, the grinder struggling to keep up on its unupgraded parts. \
-		Once the whole body entered the machine, it suddenly stopped working, and instead started spitting the meat back out, as if it was in reverse... \
-		The Cook looked over to see what was going on, but the slab of meat looked back, with massive eyes.",
-
-		"After a quick confiscation from the Russian Sol Government, most records of 'Beefmen' have gone dark, \
-		with minor glimpses of 'Sleep Experiments' going around. \
-		One thing is certain though, a rise of gibbers on-board Russian stations, which was followed by a rise in Beefmen. \
-		No one knows what happened during this era, but when the program finally ended and they were allowed into society, \
-		none of them were able to live 'normally'. Their complete inability to sleep, but their power in traversing the 'Phobetor Tear' \
-		immediately started popping everywhere, and has been at the	forefront of galaxy-wide investigations of their anatomy and the experiments.",
+		"In the year 2437 the Post-Neo-Soviet Fiddler Diner was serving food aboard the cruise ship Mustai Karim. Shortly into lunch the diner's kitchen ran out of burger meat. \
+		After straws were drawn, a neophyte cook— blissfully unaware of what would soon transpire— went to fetch some meat from the morgue. \
+		The cook thought, \"We do this all the time, right? The dead don't need their bodies, it's only being resourceful...\" \
+		Inattentively grabbing the first body they could find, the cook rushed to complete their unenviable task lest it were to become an enduring memory. \
+		What the cook didn't noticed was a green, lit revivability indicator on the morgue tray they were ransacking. The body had a soul— one that was determined to not have its vessel gibbed.",
+
+		"The young cook returned to their diner's cold room and shoved the corpse into the gibber. The machine struggled in tedium with its unupgraded parts. \
+		Once the whole body was in, the grinder's blades suddenly stopped and (with a horrible shriek) they began spinning in reverse. \
+		The cook looked over on impulse, but to their horror a grinning slab of meat looked back at them with lively yet somnambulant eyes.",
+
+		"The ship quickly reported the incident to the Third Soviet Coast Guard. Within two hours the victim had been taken by the government, and all loose ends aboard the vessal were immediately silenced. \
```

Comment:
It's lame that this is spelled out to the player, leaving it to them to think about how it went down is much better

Reply:
Changed to be a bit less direct.